Commands/Monitor.php

```diff
@@ -35,6 +36,19 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $systemCheck->checkRedisIsInstalled();
         }
 
+        $iterations = $input->getOption('iterations');
+        if (empty($iterations) && $iterations !== 0 && $iterations !== '0') {
+            $queueId = null;
```

Comment:
Is this maybe a typo? should it be `$iterations = null`?

Reply:
I wasn't too sure(I'm not too well versed on PHP); I just coppied the cade from the [`:process`](https://github.com/matomo-org/plugin-QueuedTracking/blob/master/Commands/Process.php#L45-L53) commands argument handling. I'll fix it up though.